delfin/task_manager/scheduler/schedule_manager.py

```diff
@@ -90,6 +131,13 @@ def schedule_boot_jobs(self):
                                        'PerfJobManager',
                                coordination=True)
         service.serve(job_generator)
+        partitioner = ConsistentHashing()
```

Comment:
Generic comment  SCHEDULER_BOOT_JOBS loop can be removed now..

Reply:
Yes, I'll find a place to handle the job remove and the loop would be removed